admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -21,6 +21,31 @@ place to store packages for multiple operating systems. Packages for both Ubuntu
 Ubuntu 16.04 (Xenial) and Ubuntu 18.04 (Bionic) installation instructions
 =========================================================================
 
+1. While Archivematica 1.7 is in development, please use these commands to
+   install the development repositories (**and delete this section when the
+   final release packages are made.**):
+
+   Ubuntu 16.04 (Xenial):
+
+   .. code:: bash
+
+      sudo wget -O - https://packages.archivematica.org/1.8.x/key.asc | sudo apt-key add -
+      sudo wget -O - http://jenkins-ci.archivematica.org/repos/devel.key | sudo apt-key add -
+      sudo sh -c 'echo "deb [trusted=yes] http://jenkins-ci.archivematica.org/repos/am-packbuild/1.8.0/xenial/ ./" >> /etc/apt/sources.list'
+      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.8.x/ubuntu-externals xenial main" >> /etc/apt/sources.list'
+
+   Ubuntu 18.04 (Bionic):
+
+   .. code:: bash
+
+      sudo wget -O - https://packages.archivematica.org/1.8.x/key.asc | sudo apt-key add -
+      sudo wget -O - http://jenkins-ci.archivematica.org/repos/devel.key | sudo apt-key add -
+      sudo sh -c 'echo "deb [trusted=yes] http://jenkins-ci.archivematica.org/repos/am-packbuild/1.8.0/bionic/ ./" >> /etc/apt/sources.list'
+      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.8.x/ubuntu-externals bionic main" >> /etc/apt/sources.list'
+
+    If the release has been completed, you should use these commands to install
+   the final repositories:
```

Comment:
This line needs to move right one character so that the tests pass :smile_cat: But otherwise it's good to go, so I'll approve!

Reply:
Thanks Sara, I moved this character :)